tools/functional-tester/etcd-tester/cluster.go

```diff
@@ -110,22 +134,35 @@ func (c *cluster) Bootstrap() error {
 		}
 	}
 
-	stressers := make([]Stresser, len(clientURLs))
-	for i, u := range clientURLs {
-		s := &stresser{
-			Endpoint:       u,
-			KeySize:        c.stressKeySize,
-			KeySuffixRange: c.stressKeySuffixRange,
-			N:              200,
+	var stressers []Stresser
+	if c.v2 {
```

Comment:
yeah this will be called only if user specified `---v2` flag. I am fine with dropping v2 completely.